x/evm/types/errors.go

```diff
@@ -2,44 +2,77 @@ package types
 
 import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/vm"
 )
 
-// NOTE: We can't use 1 since that error code is reserved for internal errors.
+const (
```

Comment:
so Geth defines a set of errors (for vm exec and core errors) and we also define our own (on the evm module and the SDK. It would be good to consolidate both in some way